libs/contract/truffle-config.js

```diff
@@ -36,8 +36,6 @@ module.exports = {
         );
       },
       network_id: '*', // eslint-disable-line camelcase
-      gasPrice: 10000000009, // 10 GWei
```

Comment:
Was that your problem?

Reply:
I don't know.  I don't think so - it failed even after removing it.  It often failed on goerli, but I did have success at some point by setting that to an ungodly-high number.  However, it wasn't sustainable because I would have burnt through all my test-eth in a few transactions, and it takes 24hrs to request more.

Kovan "Just Works", and uses the same modern Proof of Authority (old Ethereum is Proof of Work, new is Authority).  There is no actually advantage to Goerli - I picked it from a hat - so...